addons/mail/models/mail_message.py

```diff
@@ -758,6 +763,11 @@ def base64_to_boundary(match):
 
         message._invalidate_documents()
 
+        if 'moderation_status' in values and values['moderation_status'] == 'pending_moderation':
```

Comment:
You can write values.get('moderation_status'). By default the returned value is None is the key is not present.

Reply:
done